systems/WikibaseClient/03-Context_and_Scope.md

```diff
@@ -22,6 +22,6 @@ See further details on this decision in the [Solution Strategy](04-Solution_Stra
 | [ChangeHandler](./05-Building_Block_View.md#entity-change-notifications)      | Receives notifications to update and purge pages when an Item is changed on the [Repo](../overview/12-Glossary.md#wikibase-repository)                                                                        |   |
 | [UpdateRepoHookHandler](./05-Building_Block_View.md#linked-site-page-changes) | Notifies the [Wikibase Repository](../overview/12-Glossary.md#wikibase-repository) to update the connected Item when a page is moved or deleted                                                               |   |
 | [Data Bridge](./05-Building_Block_View.md#data-bridge)                        | A frontend component enabling [Repository](../overview/12-Glossary.md#wikibase-repository) edits on the Client via the Repository API                                                                         |   |
-| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data. For a full description see the term [Special Pages in the Glossary](../overview/12-Glossary.md#special-page)                                                    |
+| [Special Pages](./05-Building_Block_View.md#special-pages)                    | User interfaces for viewing Entity meta data. For a full description see the term [Special Pages in the Glossary](../overview/12-Glossary.md#special-page)                                                    |   |
```

Comment:
:laughing:  that was bothering me too

Reply:
the magic pre commit linting poweerrsss

Reply:
The merge in from main actually gets rid of this change now as the file was touched elsewhere :)